slides/week5/week5.org

```diff
@@ -4,19 +4,81 @@
 
 * What are we doing today?
 - Classes in C++
-- Handling merge conflicts in =git=
```

Comment:
I think this line was deleted by mistake. FWIW, you can put entries like this:

```
* TODO add hardware applications
```

and org will track them for you (even with scheduled/deadline functionality, tracking sub-tasks, etc). Org was originally a planning application.